### ISSUE TITLE
`RuleAssertionMessage` ignores the assertion inside the loop

### ISSUE THREAD

**Issue by c71n93** [volodya-lombrozo/jtcop#353]
@volodya-lombrozo I faced with some problem. 
Consider this example:
```java
    @Test
    void choosesTransformerFactoryInConcurrentEnvironment() {
        for (final Class<? extends TransformerFactory> clazz : IntStream.range(0, 100).parallel()
            .mapToObj(i -> TransformerFactory.newInstance().getClass())
            .collect(Collectors.toList())) {
            MatcherAssert.assertThat(
                "TO ADD ASSERTION MESSAGE",
                clazz,
                Matchers.typeCompatibleWith(TransformerFactoryImpl.class)
            );
        }
    }
```
For this test I have such `jtcop` output:
```
[ERROR]         2) Method choosesTransformerFactoryInConcurrentEnvironment doesn't have assertion statements.
[ERROR]         Please add at least one assertion statement to the test method.
[ERROR]         You can also ignore the rule by adding @SuppressWarnings("JTCOP.RuleAssertionMessage") annotation.
[ERROR]         Rule: RuleAssertionMessage.
```
It looks like check ignores asserts inside the loops.

**Comment by volodya-lombrozo** [comment]
@rultor release, tag is `1.2.2`, title is `1.2.2`.

**Comment by volodya-lombrozo** [comment]
@c71n93 The bug should be fixed in the `1.2.2` version.
